packages/yoga/src/Card/web/EventCard/EventCard.jsx

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import styled, { withTheme, css } from 'styled-components';
+import { oneOf, shape, string } from 'prop-types';
+import { Clock } from '@gympass/yoga-icons';
+
+import Card from '../Card';
+import Text from '../../../Text';
+
+const Event = styled(Card)`
+  display: flex;
+  flex-direction: row;
+
+  width: 100%;
+  padding: 0;
+`;
+
+const EventInfo = styled.div`
+  ${({
+    theme: {
+      yoga: {
+        components: {
+          card: { event },
+        },
+      },
+    },
+  }) => css`
+    padding: ${event.info.padding}px;
+
+    ${Text.H4} {
+      display: -webkit-inline-box;
+      height: ${event.info.name.height}px;
+      overflow: hidden;
+      margin-bottom: ${event.info.name.marginBottom}px;
+
+      font-weight: ${event.info.name.fontWeight};
+      -webkit-line-clamp: 1;
+      -webkit-box-orient: vertical;
+      xtext-overflow: ellipsis;
+      text-overflow: -o-ellipsis-lastline;
```

Comment:
What do you think to extract to something like this:

```
${Text.H4},
${Text.Small} {
  display: -webkit-inline-box;
  overflow: hidden;

  font-weight: ${event.info.fontWeight}; // <-- this was event.info.name.fontWeight, be carefull, maybe this is not right
  -webkit-line-clamp: 1;
  -webkit-box-orient: vertical;
  text-overflow: ellipsis;
  text-overflow: -o-ellipsis-lastline;
}

${Text.H4} {
  height: ${event.info.name.height}px;
  margin-bottom: ${event.info.name.marginBottom}px;
}

${Text.Small} {
  height: ${event.info.place.height}px;
  margin-bottom: ${event.info.place.marginBottom}px;
}
```

Also, it's `text-overflow`, not `xtext-overflow`.

Reply:
NICE TIP! 👍